### DISCUSSION TITLE
Is removing oneway:bicycle a bug?

### DISCUSSION THREAD

**Discussion by SLMapper** [streetcomplete/StreetComplete#4732] [Q&A] [answered]
Compare https://www.openstreetmap.org/way/50905000/history version#14 and version#15.
Previously was:
```
oneway=yes
oneway:bicycle=no
sidewalk:right:bicycle=yes
sidewalk:right:foot=designated
sidewalk:right:oneway=no
```

After setting "no cycleway" in SC `oneway:bicycle` got removed.

I am unsure if removing it is the right thing here.
I can understand the tagging as _footway + "bicycle free" sign + "both directions" sign_.
- is `sidewalk:right:oneway=no` sufficient or does it also need `oneway:bicycle`?
- should the cycleway overlay care (if it [does not care for the "bicycle free" sign](https://github.com/streetcomplete/StreetComplete/issues/4714))

**Comment by matkoniecz** [comment, marked as answer]
> Is removing oneway:bicycle a bug?

Depends on what is in reality there and what user selected as an answer. Can you share what is there as far as cycling infrastructure goes? And what you selected as an answer?

> After setting "no cycleway"

on both sides? (on oneway road in contraflow direction it should be with different text making clear that no contraflow is present)

**Reply by SLMapper** [reply]
Thank you @matkoniecz 

I am not visiting the place the next days. 
It is a split street (with oneway) for both directions and a sidewalk with footway sign and bicycle free sign. If there was an additional "in both directions" sign for the sidewalk I do not remember. 

However, after reconsidering I found that
- `bicycle:oneway` will describe the road (not the sidewalk) - _this would only be true when put on a separatly mapped sidewalk_
- there really had been a question by SC if bicycles are really not allowed in contra flow. I now actually even found the answer _none, but cyclists may use road in both directions_

Even if I did not expect SC to remove it while checking for cycleways, it seems valid and possible to avoid loosimg data. 

A bit suboptimal is that somebody cannot say that there is no cycleway without saying if bicycles are allowed in contra flow. 

No question remains to me and we can close this.

**Reply by matkoniecz** [reply]
Then I will make this as an answer :)
